entity-framework/core/modeling/inheritance.md

```diff
@@ -49,8 +46,41 @@ Finally, the discriminator can also be mapped to a regular .NET property in your
 
 [!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/NonShadowDiscriminator.cs?name=NonShadowDiscriminator&highlight=4)]
 
-## Shared columns
+### Shared columns
 
 By default, when two sibling entity types in the hierarchy have a property with the same name, they will be mapped to two separate columns. However, if their type is identical they can be mapped to the same database column:
 
 [!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/SharedTPHColumns.cs?name=SharedTPHColumns&highlight=9,13)]
+
+## Table-per-type configuration
+
+> [!NOTE]
+> The table-per-type (TPT) is a new feature in EF Core 5.0. Table-per-concrete-type (TPC) is supported by EF6, but is not yet supported by EF Core.
+
+In the TPT mapping pattern, all types are mapped to individual tables. Properties that belong solely to a base type or derived type are stored in a table that maps to that type. Tables that map to derived types also store a foreign key that joins the derived table with the base table.
+
+[!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/TPTConfiguration.cs?name=TPTConfiguration)]
+
+EF will create the following following database schema for the the model above.
+
+```sql
+CREATE TABLE [Blogs] (
+    [BlogId] int NOT NULL IDENTITY,
+    [Url] nvarchar(max) NULL,
+    CONSTRAINT [PK_Blogs] PRIMARY KEY ([BlogId])
+);
+
+CREATE TABLE [RssBlogs] (
+    [BlogId] int NOT NULL,
+    [RssUrl] nvarchar(max) NULL,
+    CONSTRAINT [PK_RssBlogs] PRIMARY KEY ([BlogId]),
+    CONSTRAINT [FK_RssBlogs_Blogs_BlogId] FOREIGN KEY ([BlogId]) REFERENCES [Blogs] ([BlogId]) ON DELETE NO ACTION
+);
+```
+
+> [!NOTE]
+> If the primary key constraint is renamed the new name will be applied to all tables mapped to the hierarchy, future EF versions will allow renaming the constraint only for a particular table.
+
+If you are employing bulk configuration you can retrieve the column name for a specific table by calling `GetColumnType`.
```

Comment:
column name from GetColumnType?

Reply:
Does it take parameter? if yes then how to get the value of parameter.
Once we had API docs updated, may be we can just reference actual API.

Reply:
The snippet below answers all your questions.
I tried linking, but it seems only types can be linked using xref

Reply:
Snippet does not use `GetColumnType` API.

xref can be used for any API which has page in .NET API explorer. What are you trying to link? GetColumnName overload which takes StoreObjectIdentifier would be missing in API explorer for now.

Reply:
I've tried linking the old GetColumnName - <xref:Microsoft.EntityFrameworkCore.RelationalPropertyExtensions.GetColumnName>

Reply:
Linked it to "all overload" page which works. In future we should update it to specific API.